### PR TITLE
Fix/update electron builder

### PR DESCRIPTION
**Pull request by SiaChenRen**
https://www.pivotaltracker.com/story/show/177487181
https://www.pivotaltracker.com/story/show/177487217

Updated the electron builder setup

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,24 +1,26 @@
 {
-  "name": "openattestation-reader",
-  "productName": "OpenAttestation Reader",
+  "name": "TradeTrust-Reader",
```
Comment (Nebulis) [inline package.json:2]: back to TT ?

Comment (SiaChenRen) [reply package.json:2]: Cause the currently opencert verifier doesn't have event listner for file upload thus i think for the time being we should remain this for tradetrust only.

Comment (Nebulis) [reply package.json:2]: openattestation !== opencerts !== tradetrust

Comment (SiaChenRen) [reply package.json:2]: I understand. As this application is made to target Tradetrust user, I plan to put the Tradetrust logo as an application icon. Kind of weird when the icon is TradeTrust but the name is OpenAttestation.

**Comment by cavacado** [comment]
discussed abit with @SiaChenRen about the changes, and will approve.

**Review by cavacado — APPROVED** [review]

**Pull request merged**